### ISSUE TITLE
TODO: Add title when i run the script on server it shows elseif command not found

### ISSUE THREAD

**Issue by engrirfan641** [koalaman/shellcheck#1177]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

A=$(pgrep mysql | wc -l)
if [ "$A" == "0" ]
echo Apache Server is not runing 
echo Restarting Apache Server
service httpd restart
exit
B=$(pgrep httpd | wc -l)
elseif [ "$B" == "0" ]
echo Apache server is not installed
then
echo Installing Apache Server
else
yum install httpd -y && service httpd start && service httpd enable

fi
```

#### Here's what shellcheck currently says:
```

```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by matthewpersico** [comment]
I'd be looking for a fatal error. Doesn't the first 'if' need a 'then'?

**Comment by dimo414** [comment]
shellcheck.net currently reports "No issues detected!" with the given script (missing `then` and all). Adding the `then` causes a few errors to show up.

Conceptually it seems like ShellCheck could catch this, but seeing as Bash won't even parse this I don't know how problematic this really is. ShellCheck can't be expected to parse [arbitrary text](https://en.wikipedia.org/wiki/Garbage_in,_garbage_out#History) and properly analyze it.

**Comment by jleffler** [comment]
There is no syntax error in the code, though it is unusual and probably unintentional shell scripting.  The `if` statement can be followed by any sequence of commands; the exit status of the command prior to the the `then` is used to establish the truth condition.  Thus in:

```
if [ "$A" == "0" ]
echo Apache Server is not runing 
echo Restarting Apache Server
service httpd restart
exit
B=$(pgrep httpd | wc -l)
elseif [ "$B" == "0" ]
echo Apache server is not installed
then
    echo Installing Apache Server
else
    yum install httpd -y && service httpd start && service httpd enable
fi
```

there is an invocation of the `test` command via its `[` synonym, followed by two `echo`s, a `service`, an `exit`, an assignment, the `elseif` command and another `echo`; and the result of the final echo (success) controls the `if` — or would if the `exit` hadn't exited the script before that is executed.  The fact that there isn't often an `elseif` command is neither here nor there as far as the official shell syntax is concerned.

Now, ShellCheck might reasonably legitimately warn that the sequence of commands after the `if` is unusual and probably erroneous, but the shell itself doesn't think there's a problem.

**Comment by matthewpersico** [comment]
As soon as I hit Comment I knew I was going to regret commenting without cut n pasting that snippet into a shell window just to see what happens. 

I would agree that it would be good if shellcheck would throw up a “yo, dude, this code is awful” warning. 

In this case “elsif” appears to be some kind of command. The fact that it is not is a runtime, somif shellcheck were to flag it, it would need to be some special case.

**Comment by koalaman** [comment]
ShellCheck already warns about `ELSE` and `else if`, but not `elseif` or `elsif`. I found several instances of this, and added a check. Thanks!
